go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/google/licenseclassifier
+
+go 1.9
+
+require (
+	github.com/google/go-cmp v0.2.0
+	github.com/sergi/go-diff v1.0.0
+	github.com/stretchr/testify v1.3.0 // indirect
```

Comment:
when I run `go mod init` and run the tests, I don't end up pulling in testify as a dependency.  Maybe there's something I missed running that pulls it in?

Reply:
`go mod tidy` adds it for me.